backend/src/main/java/com/alex/eshop/entity/Item.java

```diff
@@ -54,4 +55,11 @@ public void setImageSrc(String imageSrc) {
         this.imageSrc = imageSrc;
     }
 
-}
+    public Boolean isHasImage() {
+        return imageSrc != null && !imageSrc.isEmpty();
```

Comment:
it is not correct to return custom logic in entity/dto getter. you can do this in mapper. getter should return only field as is.